src/index.ts

```diff
@@ -94,18 +94,22 @@ export const calculateMetricsSums = (
   distance: number
   stopsCount: number
 } => {
-  return currentConnections
+  const resultingConnections = currentConnections
```

Comment:
Why is it necessary to have the same function twice? Wouldn't it make sense to add a utility function for it instead?